lib/internal/bootstrap_node.js

```diff
@@ -57,18 +57,20 @@
     if (browserGlobals) {
       // Instantiate eagerly in case the first call is under stack overflow
       // conditions where instantiation doesn't work.
-      const console = global.console;
-      console.assert;
-      console.clear;
-      console.count;
-      console.countReset;
-      console.dir;
-      console.error;
-      console.log;
-      console.time;
-      console.timeEnd;
-      console.trace;
-      console.warn;
+      if (process.config.v8_enable_inspector) {
```

Comment:
It would be nice to move that condition up in the former `if`.

Reply:
Sounds good, I'll move it. Thanks